main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -593,11 +593,12 @@ object Completion {
     *
     * @param name          the name to be completed under cursor.
     * @param qualifiedName the path of the java class we will auto-import.
+    * @param range         the range of the completion.
     * @param ap            the anchor position.
     * @param labelDetails  to show the namespace of class we are going to import
     * @param priority      the priority of the completion.
     */
-  case class AutoImportCompletion(name:String, qualifiedName: String, ap: AnchorPosition, labelDetails: CompletionItemLabelDetails, priority: Priority) extends Completion
+  case class AutoImportCompletion(name:String, qualifiedName: String, range: Range, ap: AnchorPosition, labelDetails: CompletionItemLabelDetails, priority: Priority) extends Completion
```

Comment:
Is this the right place to put range in the argument order.

Maybe it is! Just asking!

Usually we have loc: SourceLocation last

Reply:
OK; and that can be done consistently for all completions?